### PR TITLE
 swagger-codegen@2 (new formula)

### PR DESCRIPTION
**Pull request by maxliesegang**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/swagger-codegen@2.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        version: 0.0.0
+        title: Simple API
+      paths:
+        /:
+          get:
+            responses:
+              200:
+                description: OK
+    EOS
+    system "#{bin}/swagger-codegen", "generate", "-i", "minimal.yaml", "-l", "swagger"
```
Comment (igas) [inline Formula/swagger-codegen@2.rb]: Please add some assertions of output/generated files.

Comment (maxliesegang) [reply Formula/swagger-codegen@2.rb]: Thanks :) I just added an assertion for the output

**Comment by maxliesegang** [comment]
@igas Just wanted to ask if there is a chance to merge this PR or if you think this is not an acceptable versioned formula for brew-core

**Comment by SMillerDev** [comment]
Why is it needed? Is this just so people can delay updating?

**Comment by igas** [comment]
@SMillerDev I don't have anything against it, by the look of it, it's still actively maintained and not EOL. 

@maxliesegang I'll leave it for more experienced maintainers to decide.

**Comment by maxliesegang** [comment]
@SMillerDev With the bump to version 3 they lost some generators that will be migrated over time. Until that happens and the 2.x branch is still actively maintened, I think it will be useful to offer 2.x as a formula as well.

**Comment by SMillerDev** [comment]
Looks fine to me then. Just want to avoid adding versioned formula for the sake of it.

**Comment by igas** [comment]
thank you @maxliesegang!

**Pull request closed without merging**